example/lib/icons.dart

```diff
@@ -1,1625 +1,1618 @@
-  ExampleIcon(FontAwesomeIcons.weightHanging, 'weightHanging'),
-  ExampleIcon(FontAwesomeIcons.weixin, 'weixin'),
-  ExampleIcon(FontAwesomeIcons.whatsapp, 'whatsapp'),
-  ExampleIcon(FontAwesomeIcons.whatsappSquare, 'whatsappSquare'),
-  ExampleIcon(FontAwesomeIcons.wheelchair, 'wheelchair'),
-  ExampleIcon(FontAwesomeIcons.whmcs, 'whmcs'),
-  ExampleIcon(FontAwesomeIcons.wifi, 'wifi'),
-  ExampleIcon(FontAwesomeIcons.wikipediaW, 'wikipediaW'),
-  ExampleIcon(FontAwesomeIcons.wind, 'wind'),
-  ExampleIcon(FontAwesomeIcons.windowClose, 'windowClose'),
-  ExampleIcon(FontAwesomeIcons.solidWindowClose, 'solidWindowClose'),
-  ExampleIcon(FontAwesomeIcons.windowMaximize, 'windowMaximize'),
-  ExampleIcon(FontAwesomeIcons.solidWindowMaximize, 'solidWindowMaximize'),
-  ExampleIcon(FontAwesomeIcons.windowMinimize, 'windowMinimize'),
-  ExampleIcon(FontAwesomeIcons.solidWindowMinimize, 'solidWindowMinimize'),
-  ExampleIcon(FontAwesomeIcons.windowRestore, 'windowRestore'),
-  ExampleIcon(FontAwesomeIcons.solidWindowRestore, 'solidWindowRestore'),
-  ExampleIcon(FontAwesomeIcons.windows, 'windows'),
-  ExampleIcon(FontAwesomeIcons.wineBottle, 'wineBottle'),
-  ExampleIcon(FontAwesomeIcons.wineGlass, 'wineGlass'),
-  ExampleIcon(FontAwesomeIcons.wineGlassAlt, 'wineGlassAlt'),
-  ExampleIcon(FontAwesomeIcons.wix, 'wix'),
-  ExampleIcon(FontAwesomeIcons.wizardsOfTheCoast, 'wizardsOfTheCoast'),
-  ExampleIcon(FontAwesomeIcons.wodu, 'wodu'),
-  ExampleIcon(FontAwesomeIcons.wolfPackBattalion, 'wolfPackBattalion'),
-  ExampleIcon(FontAwesomeIcons.wonSign, 'wonSign'),
-  ExampleIcon(FontAwesomeIcons.wordpress, 'wordpress'),
-  ExampleIcon(FontAwesomeIcons.wordpressSimple, 'wordpressSimple'),
-  ExampleIcon(FontAwesomeIcons.wpbeginner, 'wpbeginner'),
-  ExampleIcon(FontAwesomeIcons.wpexplorer, 'wpexplorer'),
-  ExampleIcon(FontAwesomeIcons.wpforms, 'wpforms'),
-  ExampleIcon(FontAwesomeIcons.wpressr, 'wpressr'),
-  ExampleIcon(FontAwesomeIcons.wrench, 'wrench'),
-  ExampleIcon(FontAwesomeIcons.xRay, 'xRay'),
-  ExampleIcon(FontAwesomeIcons.xbox, 'xbox'),
-  ExampleIcon(FontAwesomeIcons.xing, 'xing'),
-  ExampleIcon(FontAwesomeIcons.xingSquare, 'xingSquare'),
-  ExampleIcon(FontAwesomeIcons.yCombinator, 'yCombinator'),
-  ExampleIcon(FontAwesomeIcons.yahoo, 'yahoo'),
-  ExampleIcon(FontAwesomeIcons.yammer, 'yammer'),
-  ExampleIcon(FontAwesomeIcons.yandex, 'yandex'),
-  ExampleIcon(FontAwesomeIcons.yandexInternational, 'yandexInternational'),
-  ExampleIcon(FontAwesomeIcons.yarn, 'yarn'),
-  ExampleIcon(FontAwesomeIcons.yelp, 'yelp'),
-  ExampleIcon(FontAwesomeIcons.yenSign, 'yenSign'),
-  ExampleIcon(FontAwesomeIcons.yinYang, 'yinYang'),
-  ExampleIcon(FontAwesomeIcons.yoast, 'yoast'),
-  ExampleIcon(FontAwesomeIcons.youtube, 'youtube'),
-  ExampleIcon(FontAwesomeIcons.youtubeSquare, 'youtubeSquare'),
-  ExampleIcon(FontAwesomeIcons.zhihu, 'zhihu'),
-];
+ExampleIcon(FontAwesomeIcons.fiveHundredPx, null, 'fiveHundredPx'),
```

Comment:
Indention is missing for all Icons. This makes the pub tests fail

Reply:
I'll correct the indentation

Reply:
Turns out this is related to #105 - my system cannot find `dartfmt`, either (not even using `which dartfmt`, which is weird).

I manually added some spaces to during generation to make this not an issue.